src/ffetch.js

```diff
@@ -154,6 +155,7 @@ function assignOperations(generator, context) {
     withFetch: withFetch.bind(null, generator, context),
     withHtmlParser: withHtmlParser.bind(null, generator, context),
     sheet: sheet.bind(null, generator, context),
+    getTotal: () => context.total,
```

Comment:
I would prefer a javascript getter instead of a function as this would be more natural for JS:

```
Object.assign(generator, operations, functions);
Object.defineProperty(generator, 'total', { get: () => context.total });
return generator;
```